src/server/config/__tests__/deprecation_warnings.js

```diff
@@ -0,0 +1,79 @@
+import { spawn } from 'child_process';
+
+import expect from 'expect.js';
+
+const KIBANA_WORKER = require.resolve('./fixtures/kbn_server_worker');
+const BABEL_REGISTER = require.resolve('../../../babel-register');
+const SECOND = 1000;
+
+describe('config/deprecation warnings mixin', function () {
+  this.timeout(15 * SECOND);
+
+  let stdio = '';
+  let proc = null;
+
+  before(() => new Promise((resolve, reject) => {
+    proc = spawn(process.execPath, [
```

Comment:
i've never used `spawn` in this way. does this basically fork the current process?

Reply:
Sort of, `execPath` is just a path the current node.js executable so it's a bit "safer" than just executing `node` which will have to be looked up using the `$PATH` and *could* resolve differently on different machines.